airflow/api_connexion/endpoints/connection_endpoint.py

```diff
@@ -63,11 +63,15 @@ def get_connection(connection_id, session):
 @security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_CONNECTION)])
 @format_parameters({'limit': check_limit})
 @provide_session
-def get_connections(session, limit, offset=0):
+def get_connections(session, limit, offset=0, order_by="id"):
     """Get all connection entries"""
+    to_replace = {"connection_id": "conn_id", "-connection_id": "-conn_id"}
+    allowed_filter_attrs = ['connection_id', 'conn_type', 'description', 'host', 'port', 'id']
+
     total_entries = session.query(func.count(Connection.id)).scalar()
     query = session.query(Connection)
-    connections = query.order_by(Connection.id).offset(offset).limit(limit).all()
+    query = apply_sorting(Connection, query, order_by, to_replace, allowed_filter_attrs)
```

Comment:
I feel there should be a way to avoid repeating `Connection` here. Same goes for other calls below.